orchagent/aclorch.cpp

```diff
@@ -1367,6 +1367,13 @@ bool AclTable::create()
         table_attrs.push_back(attr);
     }
 
+    if (type == ACL_TABLE_L3)
```

Comment:
will you need L3V6?

Reply:
Currently L3 table is enough. But we will need L3IPv6 in future.